libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/CloudAdapter.cs

```diff
@@ -31,8 +31,9 @@ public class CloudAdapter : CloudAdapterBase, IBotFrameworkHttpAdapter
         /// <summary>
         /// Initializes a new instance of the <see cref="CloudAdapter"/> class. (Public cloud. No auth. For testing.)
         /// </summary>
-        public CloudAdapter()
-            : this(BotFrameworkAuthenticationFactory.Create())
+        /// <param name="config">The <see cref="IConfiguration"/> instance to use for adapter configuration.</param>
+        public CloudAdapter(IConfiguration config)
```

Comment:
This is breaking things, is it possible to preserve the empty constructor and create an overload for this?

Reply:
since the CloudAdapter configured in DI, and CreateBuilder already injects IConfiguration we should be good here. no?